extensions/php/languages/php/runnables.scm

```diff
@@ -0,0 +1,61 @@
+; Class that follow the naming convention of PHPUnit test classes
+; and that doesn't have the abstract modifier
+; and have a method that follow the naming convention of PHPUnit test methods
+; and the method is public
+(class_declaration
+    modifier: (_)? @modifier
+    (#not-eq? @modifier "abstract")
+    name: (_) @name
+    (#match? @name ".*Test$")
+    body: (declaration_list
+        (method_declaration
+            (visibility_modifier)? @visibility
+            (#eq? @visibility "public")
+            name: (_) @run
+            (#match? @run "^test.*")
+        )
+    )
+) @phpunit-test
+
+; Class that follow the naming convention of PHPUnit test classes
+; and that doesn't have the abstract modifier
+; and have a method that has the @test annotation
+; and the method is public
+(class_declaration
+    modifier: (_)? @modifier
+    (#not-eq? @modifier "abstract")
+    name: (_) @name
+    (#match? @name ".*Test$")
+    body: (declaration_list
+        ((comment) @comment
+            (#match? @comment ".*@test\\b.*")
+        .
+        (method_declaration
+            (visibility_modifier)? @visibility
+            (#eq? @visibility "public")
+            name: (_) @run
+            (#not-match? @run "^test.*")
+        ))
+    )
+) @phpunit-test
```

Comment:
I hope my mentioning Pest didn't make this more complicated than it needed to be! I'm sorry if it did. I've not used pest, but as far as the phpunit support, this LGTM!

> agree that \b it better is in this case

Yes, I think that `\b` is the way to go! Good call 👍 

> deprecate docblocks in PHP and just use attributes

So ... I wasn't aware of this ... and it's definitely something that can be left for a followup PR to not further complicate this ... *but* ... your comment sent me looking and apparently phpunit does have YAWTMAT (yet another way to mark a test), using attributes: https://docs.phpunit.de/en/11.1/attributes.html#test I've not used it (wasn't even aware of it!) but I think that adding these lines to the `method_declaration` in the other query (for `test*` methods) may work:
```lisp
attributes: (attribute_list (attribute_group (attribute (name) @attr)))?
(#eq? @attr "Test")
```

Reply:
I heard they were planning to add attribute support, but didn't know it was already out. But thanks for pointing it out! Will add support for it.

Reply:
No, it's fine!, I was already planning to make a follow-up for **Pest** support. But when you mentioned it I thought let just add it inside this PR. Yeah, it is more complicated than I thought, tho.